components/clp-py-utils/clp_py_utils/clp_config.py

```
@@ -401,9 +401,10 @@ def load_credentials_from_env(self, user_type: ClpDbUserType = ClpDbUserType.CLP
             password=_get_env_var(pass_env_var),
         )
 
-    def transform_for_container(self):
-        self.host = DB_COMPONENT_NAME
-        self.port = self.DEFAULT_PORT
+    def transform_for_container(self, is_bundled: bool):
```

Ruff (0.15.1)

[error] Boolean-typed positional argument in function definition

(FBT001)